test/test_pipeline/components/regression/test_base.py

```diff
@@ -286,3 +293,138 @@ def test_module_idempotent(self):
                     seed == random_state
                     for random_state in [rs_1, rs_estimator_1, rs_2, rs_estimator_2]
                 ])
+
+
+@pytest.mark.parametrize("regressor", _regressors.values())
+@pytest.mark.parametrize("X", [np.array([[1, 2, 3]] * 20)])
```

Comment:
Just checking, X and y are just one value and so there's no need to have them as parameters, right?

Reply:
Not really but it's just a personal taste, if in the future we find we need different kinds of values, it's easy to add. It also keeps the test function to testing and not for defining anything to set up the test.